### DISCUSSION TITLE
`gh-deploy` doesn't deploy with `mkdocs-material`

### DISCUSSION THREAD

**Discussion by asciidude** [mkdocs/mkdocs#3002] [Q&A]
![image](https://user-images.githubusercontent.com/39356292/195557907-8e3ab18e-e282-487c-9926-455dadda2953.png)

`mkdocs.yml`
```js
site_name: Atlis API
theme:
  name: material

  logo: Assets/icon.png

  palette:
    # Palette toggle for light mode
    - media: "(prefers-color-scheme: light)"
      scheme: default
      toggle:
        icon: material/brightness-7
        name: 🌚 Shut the curtains

    # Palette toggle for dark mode
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      toggle:
        icon: material/brightness-4
        name: 🌞 Let the sun in

plugins:
  - search
  - i18n:
      default_language: en
      languages:
        en: English
        nl: Nederlands

markdown_extensions:
  - attr_list
  - pymdownx.escapeall:
      hardbreak: true
```

I haven't tested it with any other themes, but it does work with the default theme

**Comment by asciidude** [comment]
Bump

**Comment by AgedLace** [comment]


**Reply by oprypin** [reply]
> I've created the index.html in the root of the project 

I think that's likely the problem. MkDocs doesn't look at html files, so indeed it's expected that you see it unchanged. You should create *.md files instead.

Also please don't respond to someone else's issue with some other different issue.

**Comment by Andre601** [comment]
I personally suggest to ask this in the actual [MkDocs Material](https://github.com/squidfunk/mkdocs-material) repository since A) It's not a default theme included in MkDocs and B) this is perhaps an issue with Material for MkDocs.
